src/fagfunksjoner/data/view_dataframe.py

```diff
@@ -8,36 +8,54 @@
 def filter_display(
     dataframe: pd.DataFrame,
     column: str,
-    value: widgets.Widget,
-    operator: widgets.Dropdown,
+    value: str | int | float | tuple[str | int | float, ...],
+    operator: str,
 ) -> None:
     """Filter data based on args, and display the result.
 
     Args:
         dataframe (pd.DataFrame): The DataFrame to filter.
         column (str): Column to base filter on.
-        value (widgets.Widget): Widget containing the value to compare filter against.
-        operator (widgets.Dropdown): Widget containing how to compare column against value.
-    """
-    # Extract the actual value from the widgets
-    filter_value = value.value
-    filter_operator = operator.value
+        value (str | int | float):The value to compare filter against.
```

Comment:
You don't need to repeat the argument types in the docstring when you are using type hints. The types are extracted from the function declaration when the documentation is generated. This saves time and avoids possible mismatches, like the one for the value parameter here. See examples here: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Reply:
Ok, guess just an unneccessary habit. Will work on it going forward ;)